diff -r 54ce1642efe9 file_diff/Makefile
--- a/file_diff/Makefile	Sat Jun 20 23:21:48 2020 -0400
+++ b/file_diff/Makefile	Mon Jun 22 06:49:28 2020 +0200
@@ -56,3 +56,9 @@
 $(diff_match_patch_zip):
 	wget https://github.com/leutloff/diff-match-patch-cpp-stl/archive/$@
 diff_match_patch.h: | $(diff_match_patch_zip) ; unzip -j $| "*/$@"
+
+install:
+	$(INSTALL) -d $(DESTDIR)$(PREFIX)/share/textadept/modules/filediff
+	$(INSTALL) -d $(DESTDIR)$(PREFIX)/share/doc/textadept-modules-filediff
+	$(INSTALL) *.lua *.so $(DESTDIR)$(PREFIX)/share/textadept/modules/filediff
+
