diff -r 66e09cf24057 yaml/src/Makefile
--- a/yaml/src/Makefile	Thu Jul 06 10:16:15 2017 -0400
+++ b/yaml/src/Makefile	Mon Jan 08 07:00:26 2018 +0100
@@ -15,12 +15,16 @@
            -undefined dynamic_lookup
   target = ../libyamlosx.so
 else
+  LIBYAML=libyaml$(findstring 64, $(findstring amd64,$(shell dpkg-architecture -q DEB_TARGET_ARCH))).so
   CC = gcc
   CFLAGS = -fPIC
   LDFLAGS = -Wl,--retain-symbols-file -Wl,lua.sym
-  target = ../libyaml.so
+  target = ../$(LIBYAML)
 endif

+PREFIX ?= /usr/local
+INSTALL ?= install -m 755 -o root -g root
+
 yaml_objs = api.o dumper.o emitter.o loader.o parser.o reader.o scanner.o \
             writer.o
 lua_objs = lemitter.o lparser.o lscanner.o lyaml.o
@@ -51,3 +55,10 @@
 	$(CROSS)$(CC) -shared -o $@ $^ -L$(ta_src) -llua51
 clean:
 	rm -f *.o
+	rm -f ../*.dll ../*.so
+
+install:
+	$(INSTALL) -d $(DESTDIR)$(PREFIX)/share/textadept/modules/yaml
+	$(INSTALL) -d $(DESTDIR)$(PREFIX)/share/doc/textadept-modules-yaml
+	$(INSTALL) ../*.lua ../*.so $(DESTDIR)$(PREFIX)/share/textadept/modules/yaml
+	$(INSTALL) LICENSE.*  $(DESTDIR)$(PREFIX)/share/doc/textadept-modules-yaml
